app/resources/brave_generated_resources_en-GB.xtb

```diff
@@ -335,4 +335,4 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
 <translation id="2808630562319398131">Reset Brave Rewards data ...</translation>
 <translation id="1224064034972923114">Help improve Brave by automatically sending crash reports</translation>
 <translation id="3738833060602717053">Help improve <ph name="PRODUCT_NAME"/> by automatically sending crash reports</translation>
-</translationbundle>
\ No newline at end of file
+</translationbundle>
```

Comment:
@deepxcode can you please revert this line, thanks

Reply:
I don't know why this is showing as a change, I haven't change this line at all in this commit.
Also, 
It shows I have deleted `</translationbundle>` and added `</translationbundle>` , Which turns out to the same thing. I guess, there will be no issue in this.

Reply:
no worries; it's just a different line ending basically

Reply:
Fixed 😄 I did a rebase, took your first commit, and dropped the others (since first commit solved it perfectly!)

Reply:
Oh okay, Thank you @bsclifton. Also, how did you rebase and dropped other commits of mine?

Reply:
@deeppandya Basically I switched to your branch and then did an interactive rebase (`rebase -i origin/master`) against master

When you do this, the list of commits will show in vim (or whatever editor you have setup) and you can take an action on them. I kept your first commit and then just deleted the second two (you can either delete the line or add a `d` in front of the line instead of `p` (pick)). Then I did a force push to your remote 😄

Reply:
Oh okay, got it! Thank you😄
Also, you mentioned a wrong user😅